src/SW_Output_outtext.c

```diff
@@ -168,47 +166,48 @@ static void create_csv_headers(
                     goto freeMem; // Exit function prematurely due to error
                 }
 
-                helpBufFull = sw_memccpy_inc(
+                fullBuffer = sw_memccpy_inc(
                     (void **) &writePtrHelp,
+                    endHelpPtr,
                     (void *) str_help1,
                     '\0',
                     &writeSizeHelp
                 );
-                if (helpBufFull) {
+                if (fullBuffer) {
                     goto freeMem;
                 }
+                printf("%s %zu\n", str_help1, writeSizeHelp);

```

Comment:
create_csv_headers(): please remove the debug/development statement `printf("%s %zu\n", str_help1, writeSizeHelp);`